tla-io/src/main/scala/at/forsyte/apalache/tla/imp/SanyImporter.scala

```diff
@@ -68,8 +68,14 @@ class SanyImporter(sourceStore: SourceStore, annotationStore: AnnotationStore) e
     // Resolver for filenames, patched for wired modules.
     val filenameResolver = SanyNameToStream(libraryPaths)
 
+    // Set a unique tmpdir to avoid race-condition in SANY
+    // TODO: RM once https://github.com/tlaplus/tlaplus/issues/688 is fixed
+    //       see
```

Comment:
I think the second line should either go or reference an Apalache issue?

```suggestion
    // TODO: RM once https://github.com/tlaplus/tlaplus/issues/688 is fixed
```

Reply:
It looks like [createTempDirectory](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory(java.lang.String,%20java.nio.file.attribute.FileAttribute...)) does not need a pattern, just a prefix.

Reply:
> It looks like [createTempDirectory](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory(java.lang.String,%20java.nio.file.attribute.FileAttribute...)) does not need a pattern, just a prefix.

If this is in response to my comment above, I meant `XXX` and `YYY` to be wildcards, not patterns to `createTempDirectory`.